index.js

```diff
@@ -1011,10 +1022,13 @@
   //.
   //. > S.Just(S.Right(42)).sequence(S.Either.of)
   //. Right(Just(42))
+  //.
+  //. > S.Just(S.Left('Cannot divide by zero')).sequence(S.Either.of)
+  //. Left('Cannot divide by zero')
   //. ```
   Maybe.prototype.sequence =
   method('Maybe#sequence',
-         {a: [Apply], b: [Apply]},
+         {a: [Applicative], b: [Applicative]},
```

Comment:
It's arguably correct either way, given that the function to make the `Apply` and `Applicative` is explicitly provided by the caller. That said, I think it think it probably makes more said using `Applicative` here.